README.md

```diff
@@ -4,7 +4,7 @@
 [![Code Health](https://landscape.io/github/mushorg/conpot/master/landscape.png)](https://landscape.io/github/mushorg/conpot/master)
 [![Python Version](https://img.shields.io/pypi/pyversions/conpot.svg)](https://pypi.python.org/pypi/Conpot) 
```

Comment:
I think this badge is broken. At least it's not showing anything useful.

Reply:
Yeah. Strange. It would've been useful to know what versions of python Conpot supports. 
I thought I got this right. 

[I think this updates from Pypi](https://stackoverflow.com/a/47315913/2873994). So this should update when there is bump in version.

Would it be okay to comment this? and perhaps later uncomment when the version at pypi is updated?

Reply:
We can leave it in there and see if it updates on a new pypi release